apps/web/vibes/soul/form/button-radio-group/index.tsx

```diff
@@ -43,6 +44,11 @@ export const ButtonRadioGroup = React.forwardRef<
             disabled={option.disabled}
             id={option.value}
             key={option.value}
+            onMouseEnter={() => {
+              if (typeof onOptionMouseEnter === 'function') {
+                onOptionMouseEnter(option.value);
+              }
+            }}
```

Comment:
```suggestion
            onMouseEnter={() => {
              onOptionMouseEnter?.(option.value);
            }}
```

I suggest we apply that change to all instances of this pattern

Reply:
I was trying to be more type safe but this is fine by me.